### ISSUE TITLE
Build repo showing failure on unknown environment

### ISSUE THREAD

**Issue by privateducky** [preludeorg/build#389]
Who am I?

![Screenshot 2023-01-21 at 3 11 30 PM](https://user-images.githubusercontent.com/49954156/213885394-1a1fe1df-6d37-456a-8f65-0a0712e684b6.png)

**Comment by kenrick** [comment]
@packplusplus could you look into this one?

**Comment by packplusplus** [comment]
tldr; Kind of an impossible to fix thing, related to a bunk github action event, can't really clean it up.

Github actions can specify an "environment" for your job to source secrets. I thought you could reference a variable set in the workflow file to specify the environment. I got the syntax wrong and did `environment: env.ENVIRONMENT` instead of `environment: ${{ env.ENVIRONMENT }}`. Github saw fit to create a new environment in it's history for this worflow. I've since deleted that workflow run, but the internet forgets nothing.

To completely close the loop on this, variables aren't available to set the `environment` anyways, so when I later tried to use `environment: ${{ env.ENVIRONMENT }}` I got a more proper error, and github actions didn't add that as an "environment" to history.
